notebooks/222-vision-image-colorization/README.md

```diff
@@ -0,0 +1,13 @@
+# Image Colorization with OpenVINO Tutorial
+
+![Let there be color](https://user-images.githubusercontent.com/18904157/180923280-9caefaf1-742b-4d2f-8943-5d4a6126e2fc.png)
+
+The idea of colorization is given a grayscale image as input, the model hallucinates a plausible, vibrant & realistic colorized version of the image.
+
```

Comment:
```suggestion
"#### About Colorization-v2\n",
    "* The colorization-v2 model is one of the colorization group of models designed to perform image colorization.\n",
    "* Model was trained on ImageNet dataset.\n",
    "* Model consumes as input L-channel of LAB-image and give as output predict A- and B-channels of LAB-image.\n",
    "\n",
    "#### About Colorization-siggraph\n",
    "* The colorization-siggraph model is one of the colorization group of models designed to real-time user-guided image colorization.\n",
    "* Model was trained on ImageNet dataset with synthetically generated user interaction.\n",
    "* Model consumes as input L-channel of LAB-image and yields output predict A- and B-channels of LAB-image.\n",
    "\n",
    "Check out [colorization](https://github.com/richzhang/colorization) repository for more details."
```

Reply:
@Davidportlouis We need to improve a little bit the contextualization of the notebook. Could you please review my suggestion? Thanks

Reply:
@Davidportlouis please add a better contextualization of the problem in the readme file. As this 
![image](https://user-images.githubusercontent.com/10940214/181347202-2ec3d353-f14d-49c2-b935-492731cdcaea.png)

Reply:
@Davidportlouis Please add these paragraphs to the readme file in your notebook folder.

![image](https://user-images.githubusercontent.com/10940214/181490868-aca00d4b-ba78-4c8d-a821-b7378c8489ce.png)